### DISCUSSION TITLE
decreasing page load time of table using wire:init ?

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#783] [Ideas]
why not havving something like this in our table body to show the page even tho if the data is not there yet to decrease page time load specially for huge records
```
<div wire:init="fetchDatasource">
  @if($Datasource)
     <x-table.body data="{{ $Datasource }}" />
  @else
  	Loading data
  @endif
</div>```

**Comment by luanfreitasdev** [comment]
I think it's a good one

**Reply by BenOussama180** [reply]
thanks, i got inspired by freek's recent tweet , hope it'll make it to v4!

**Reply by luanfreitasdev** [reply]
https://github.com/Power-Components/livewire-powergrid/pull/790

**Reply by BenOussama180** [reply]
Glad to see this comming to life, thanks alot, cant wait for the V4
